### PR TITLE
AMQStreams: enabling cruise-control

### PR DESCRIPTION
**Pull request by afsanjar**
Required by AMQStream

### CODE REVIEW

**Review by krishvoor — COMMENTED** [review]
Hi Amir,
Can you kindly add the Apache2 LICENSE File, please?

File: c/cruise-control/redhat_ubi8/build.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+#!/bin/bash
+if [ -z $1 ] 
+then
+	# Master branch
+	BRANCH=""
+else
+	BRANCH="--branch "$1
+fi
+BUILD_DIR=""cruise-control
```
Comment (Abhijit-Mane) [inline c/cruise-control/redhat_ubi8/build.sh:26]: Is this a type and you instead meant  BUILD_DIR=**"cruise-control"**  ?

Comment (afsanjar) [reply c/cruise-control/redhat_ubi8/build.sh]: good catch, done

**Review by Abhijit-Mane — COMMENTED** [review]
Looks good to me.

File: c/cruise-control/redhat_ubi8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM registry.access.redhat.com/ubi8/ubi
```
Comment (gerrith3) [inline c/cruise-control/redhat_ubi8/Dockerfile:1]: Why not use a base of ubi8/openjdk-8 here instead of installing it separately?  Should be faster.

**Review by gerrith3 — CHANGES_REQUESTED** [review]
Lmk if the updated ubi base image doesn't work, but I think that should simplify things just a tiny bit.  It probably also will protect you from CVE exposure by not fixing the OpenJDK maintenance version and it should be handled by Red Hat updated that ubi base image.

File: c/cruise-control/redhat_ubi8/Dockerfile (hunk 2 above)
Comment (afsanjar) [reply c/cruise-control/redhat_ubi8/Dockerfile:1]: Correct @gerrith3, but as you might recall I reported two weeks ago that ubi8/openjdk-8 image was not working, it had JBOS as the user and not root.

**Review by Abhijit-Mane — APPROVED** [review]
All changes incorporated, looks okay to me.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**